### ISSUE TITLE
"DDEV Local Roadmap" outdated / obsolete

### ISSUE THREAD

**Issue by tyler36** [drud/ddev#3325]
**Describe the bug**

The DDEV Local Roadmap was last updated 2020-11-24. 

It is:
- missing 1.18 release information
- missing proper 1.17 release information
- missing a release date for v1.16 (all other release include this)

Because it's a Github Wiki page, I don't think I am able to update it myself.

**Expected behavior**

To be honest, I've never clicked on a Github Wiki page. I think the link came up in search while trying to debug an issue. However, it's obviously

If it is no longer used, we should probably remove it ( and references to it).

If it still relavant, could we please make the following changes:
- [ ] add missing 1.18 release information
- [ ] add proper 1.17 release information (mark checkboxes, list included changes etc...)
- [ ] add missing a release date for v1.16
- [ ] order historical releases in descending order

Descending order for historical releases should be used because
- recent changes are more relevant for current users 
- I need to scroll to find recent changes and support (EG. PHP 8)
- it maintains the same order as "[releases](https://github.com/drud/ddev/releases)" page
- a "2021-xxx" date leaves me with an "active repo" impression, seeing "2019-06-26" make me feel like it's outdated

Personal preference perhaps?

**Comment by tyler36** [comment]
>  you may already have edit privileges on it

I don't think I have edit privileges. I looked at one of my public repos (Wikis aren't available on private repos) and there is a `Edit` & `New Page` button.
These are not present on the [DDEV/DDEV wiki](https://github.com/drud/ddev/wiki/DDEV-Local-Roadmap).  The "Home" wiki was update by "Brad Bowman" in 2017 so they must be/have been the owner or collaborator on the project.

The DDEV wiki seems to contain 3 pages
- "[Home](https://github.com/drud/ddev/wiki)" not used at all
- "[DDEV Local Previous Releases](https://github.com/drud/ddev/wiki/DDEV-Local-Previous-Releases)" which has some value
- "[DDEV Local Roadmap](https://github.com/drud/ddev/wiki/DDEV-Local-Roadmap)" which inspired the OP

There is also a `[DDEV-Local Roadmap](https://github.com/drud/ddev/wiki/wiki/DDEV-Local-Roadmap)` area below the pages which confusingly sends me to the "Home"
![DDEV-wiki](https://user-images.githubusercontent.com/7234392/138373830-47681a61-863a-4154-8216-eab8237af0ea.jpg)
 page with a "You do not have permission to update this wiki." error.

**Comment by rfay** [comment]
The one thing about the historical releases is it's really hard to parse  https://github.com/drud/ddev/releases because of all the prereleases there. So understanding the history of the project is not easy that way.

**Comment by tyler36** [comment]
Just had a quick search. Cloning a github repo doesn't clone the wiki. I can clone just the wiki page but I don't have permission to push to it.

Here is an updated roadmap page:
- sort historic items in decending order
- fixed heading levels
- add missing data for version 1.16
- various styling fixes



[DDEV-Local-Roadmap.md](https://github.com/drud/ddev/files/7393567/DDEV-Local-Roadmap.md)

**Comment by tyler36** [comment]
For reference:

> [About Wiki](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis)
You can edit wikis directly on GitHub, or you can edit wiki files locally. By default, only people with write access to your repository can make changes to wikis, although you can allow everyone on GitHub.com to contribute to a wiki in a public repository. For more information, see "[Changing access permissions for wikis](https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis)".

**Comment by rfay** [comment]
You should be able to check out the wiki if that helps, but we should lose the wiki anyway. A PR putting that stuff into docs/users/topics/release_history.md would be lovely.

**Comment by tyler36** [comment]
Not sure how to "fork" a wiki. I checked it out and made changes but can't upload them.

**Comment by rfay** [comment]
Right, you don't need to fork it. Create a PR against ddev docs, with those contents, putting the stuff at docs/users/topics/release_history.md

**Comment by tyler36** [comment]
Hmmm ... I usually do pull request via VScode but it shows as "No remotes match the current setting"

**Comment by tyler36** [comment]
Sorry ... I can't read. Was trying to update the `wiki.git` ...
